internal/ingress/controller/nginx.go

```diff
@@ -602,6 +602,7 @@ func (n *NGINXController) OnUpdate(ingressCfg ingress.Configuration) error {
 		PublishService:             n.GetPublishService(),
 		DynamicCertificatesEnabled: n.cfg.DynamicCertificatesEnabled,
 		EnableMetrics:              n.cfg.EnableMetrics,
+		ControllerPods:             ingressCfg.ControllerPods,
```

Comment:
Exposing this to template can lead to bugs since we aren't going to regenerate the configuration when its value changes, we will dynamically configure it.

Reply:
What would be the best option here? Only expose the number of running replicas or totally remove it from template? And only leave it in Lua?

Reply:
I think don't expose anything at all and have it exposed to Lua only.